Analyzer.py

```diff
@@ -315,25 +335,26 @@ def main():
         logger.debug(f" Calculating selection mask on VFAT DAQ mask")
         DAQMaskedVFAT_mask = calcDAQMaskedVFAT_mask(gemPropHit, gemOHStatus)
         muonTrack_mask["DAQMaskedVFAT"] = DAQMaskedVFAT_mask
-        muonTrack_mask["overallGood_Mask"] = muonTrack_mask["overallGood_Mask"] & DAQMaskedVFAT_mask
+        #print ("DAQMaskedVFAT_mask",  DAQMaskedVFAT_mask)
+        muonTrack_mask["overallGood_Mask"] = ak.where(input_par["gemprophit_array"].mu_propagated_station<2, muonTrack_mask["overallGood_Mask"] & DAQMaskedVFAT_mask, muonTrack_mask["overallGood_Mask"])
         heap_size(the_heap, "after calculating VFAT DAQ mask")
 
         logger.debug(f" Calculating selection mask on VFAT DAQ missing")
         DAQMissingVFAT_mask = calcDAQMissingVFAT_mask(gemPropHit, gemOHStatus)
         muonTrack_mask["DAQMissingVFAT"] = DAQMissingVFAT_mask
-        muonTrack_mask["overallGood_Mask"] = muonTrack_mask["overallGood_Mask"] & DAQMissingVFAT_mask
+        muonTrack_mask["overallGood_Mask"] = ak.where(input_par["gemprophit_array"].mu_propagated_station<2, muonTrack_mask["overallGood_Mask"] & DAQMissingVFAT_mask, muonTrack_mask["overallGood_Mask"])
         heap_size(the_heap, "after calculating VFAT DAQ missing")
 
         logger.debug(f" Calculating selection mask on DAQ error")
         DAQError_mask = calcDAQError_mask(gemPropHit, gemOHStatus)
         muonTrack_mask["DAQError"] = DAQError_mask
-        muonTrack_mask["overallGood_Mask"] = muonTrack_mask["overallGood_Mask"] & DAQError_mask
+        muonTrack_mask["overallGood_Mask"] = ak.where(input_par["gemprophit_array"].mu_propagated_station<2, muonTrack_mask["overallGood_Mask"] & DAQError_mask, muonTrack_mask["overallGood_Mask"])
         heap_size(the_heap, "after calculating DAQ error mask")
 
         logger.debug(f" Calculating selection mask on DAQ enabled OH")
         DAQenabledOH_mask = calcDAQenabledOH_mask(gemPropHit, gemOHStatus)
         muonTrack_mask["DAQenabledOH"] = DAQenabledOH_mask
-        muonTrack_mask["overallGood_Mask"] = muonTrack_mask["overallGood_Mask"] & DAQenabledOH_mask
+        muonTrack_mask["overallGood_Mask"] = ak.where(input_par["gemprophit_array"].mu_propagated_station<2, muonTrack_mask["overallGood_Mask"] & DAQenabledOH_mask, muonTrack_mask["overallGood_Mask"])
```

Comment:
Why don't you want to apply any DAQ related mask for GE21?

Reply:
Because I don't understand how the DAQ masks are calculated at the moment, so I don't know how to modify that code for GE21. 
Also, fortunately the GE21 electronics at the moment is much more stable than GE11.

Reply:
Ok we can keep it this way for now